VWOSdk/Core/VWOClient.cs

```diff
@@ -17,6 +17,7 @@
 #pragma warning restore 1587
 
 using System.Collections.Generic;
+using System;

```

Comment:
do we really need System , can we not import specific namespace?

Reply:
Not anymore, need it for console. shall remove now.